core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -246,24 +329,36 @@ private boolean isIntermediateTopic(final String topic) {
         return options.valuesOf(intermediateTopicsOption).contains(topic);
     }
 
-    private void deleteInternalTopics(final ZkUtils zkUtils) {
-        System.out.println("Deleting all internal/auto-created topics for application " + options.valueOf(applicationIdOption));
+    private void maybedeleteInternalTopics(final ZkUtils zkUtils) {
+        Boolean dryRun = options.has(dryRunOption);
+
+        if (!dryRun) {
+            System.out.println("Deleting all internal/auto-created topics for application " + options.valueOf(applicationIdOption));
+        } else {
+            System.out.println("Following internal/auto-created topics will be Deleted for application " + options.valueOf(applicationIdOption));
```

Comment:
I understand the motivation and agree it's a good idea. It's only that 

`"Deleting all internal/auto-created topics for application <app.id>."`
and
`"Following internal/auto-created topics will be Deleted for application <app.id>."`

sound super similar. So not sure if we gain much. But it's quite subjective. It's ok if you leave as is. Please fix the typo though `Deleted` -> `deleted`.